design-proposals/passt-feature-gate-iface-api-deprecation.md

```diff
@@ -0,0 +1,62 @@
+## Overview
+Following the emerging [Kubevirt Network Binding Plugin API](https://docs.google.com/document/d/13fLaqZuM8El_CT5XzxPv-1epKsB1-Spv_jtXveEkkdQ/edit#heading=h.z27rudfpwy44)
+and introduction of Passt network binding plugin [2](https://github.com/kubevirt/kubevirt/pull/10425) [3](https://github.com/kubevirt/kubevirt/pull/10621), 
+it's now possible to remove Passt network binding implementation from Kubevirt core.
+
+This proposal outlines the replacement plan for the existing Passt implementation and API.
+
+# Motivation
+Removing Passt network binding implementation from Kubevirt core enable reduce the maintenance burden.
+Having Passt binding code outside Kubevirt's core may ease contribution process as there is no need to be familiar 
+with [kubevirt/kubevirt](https://github.com/kubevirt/kubevirt) core code.
+
+## Goals
+- Reduce maintenance of Passt network binding in Kubevirt core.
+  
+## Non Goals
+
+## User Stories
+
+## Repos
+[kubevirt/kubevirt](https://github.com/kubevirt/kubevirt):
+- [Passt binding plugin hook sidecar](https://github.com/kubevirt/kubevirt/tree/main/cmd/sidecars/network-passt-binding)
+- [Passt binding plugin CNI](https://github.com/kubevirt/kubevirt/tree/main/cmd/cniplugins/passt-binding)
+
+# Design
+Replace Kubevirt core Passt network binding implementation and interface API with Passt binding plugin.
+Users should be aware of the coming changes and be prepared.
+
+The proposed plan for replacing Kubevirt core Passt binding implementation and API is as follows:
+
+1. Announce the coming changes for Passt binding in kubevirt-dev mailing list.
```

Comment:
The "changes" are not known yet at this stage, either add a short description of what they are (e.g. deprecation and removal of passt core binding feature) or a ref to where it is defined.